docs/ccdproc/image_management.rst

```diff
@@ -25,6 +25,10 @@ example initialization looks like::
     >>> keys = ['imagetyp', 'object', 'filter', 'exposure']
     >>> ic1 = ImageFileCollection('.', keywords=keys) # only keep track of keys
 
+    >>> # the following line shouldn't be needed in real files but it's needed
+    >>> # because the testfile has no header entry for "filter".
+    >>> ic1.summary['filter'] = ic1.summary['filter'].astype(str)
+
```

Comment:
if we need to for now, yes....to summarize, as I understand it, the sort fails now because of a change in `np.ma.argsort`. I think we should open a separate issue for that, because it will conceivably affect real use cases.